monitoring/grafana/bin/save-dashboards.sh

```diff
@@ -27,7 +27,8 @@ CLEAR_DASHBOARD_ID='.id = null'
 
 # CLEAR PROMETHEUS UID
 # clears prometheus uid since each deployment is unique
-CLEAR_PROM_UID='del(.panels[].targets[]?.datasource.uid)'
+CLEAR_PROM_TARGET_UID='del(.panels[].targets[]?.datasource.uid)'
```

Comment:
not part of the scope of this feature but have you seen this? https://github.com/mpostument/grafana-sync

Reply:
No, I haven't, thanks!

I added it as a comment to INF-169 since it might help with our folder support.